app/cdap/components/WranglerV2/SearchField/index.tsx

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import React from 'react';
+import { Box, Divider, Input, InputProps, Link, Typography } from '@material-ui/core';
+import CloseOutlinedIcon from '@material-ui/icons/CloseOutlined';
+import InfoOutlinedIcon from '@material-ui/icons/InfoOutlined';
+import T from 'i18n-react';
+import styled from 'styled-components';
+import { grey } from '@material-ui/core/colors';
+
+export interface IDirectiveSuggestionProps {
+  link: string;
+  usage: string;
+}
+
+interface ISearchFieldProps {
+  onPostfixIconClick: () => void;
+  inputProps: InputProps;
+  directiveSuggestion: IDirectiveSuggestionProps;
+}
+
+export const StyledInput = styled(Input)`
+  &&& {
+    width: 100%;
+    margin: 0px 5px;
+    outline: 0;
+    background: transparent;
+    color: #ffffff;
+    & .MuiInputBase-input {
+      padding: 0px;
+      height: 12px;
+      color: #ffffff;
+      font-size: 14px;
+    }
+  }
+`;
+
+const PrefixIconWrapper = styled(Box)`
+  color: #94ec98;
+`;
+
+const PostfixIconWrapper = styled(CloseOutlinedIcon)`
+  cursor: pointer;
+`;
+
+const DirectiveWrapper = styled(Box)`
+  padding: 0px 15px;
+`;
+
+const DividerLine = styled(Divider)`
+  &.MuiDivider-root {
+    background-color: ${grey[300]};
+    opacity: 0.2;
+    margin-bottom: 10px;
+  }
+`;
+
+const InfoLink = styled(Link)`
+  &&& {
```

Comment:
we might not need higher specificty &&&
plz check and remove if not required across all styles

Reply:
ok